pxr/usdImaging/bin/testusdview/CMakeLists.txt

```diff
@@ -331,9 +432,253 @@ if (NOT PXR_HEADLESS_TEST_MODE)
         EXPECTED_RETURN_CODE 0
     )
 
+    pxr_register_test(testUsdviewFreeCamera
+        PYTHON
+        COMMAND "${CMAKE_INSTALL_PREFIX}/bin/testusdview --testScript testUsdviewFreeCamera.py Block.usda"
+        IMAGE_DIFF_COMPARE
+            start.png
+            block_F.png
+            block_L.png
+            block_B.png
+            block_R.png
+            phi45.png
+            adjustDist_05.png
+            adjustDist_05_20.png
+            rot_truck.png
+            truck_-1_-1.png
+            truck_1_1.png
+            end.png
+        FAIL 1
+        FAIL_PERCENT 5
```

Comment:
I think the initial values for these fail percentages are too high. I'm guessing you got those from the perceptualdiff calls in the testSpecs.xml, but the tolerance numbers there are # of pixels, not percentages. It'd be great if we could get values that are more representative here.

Reply:
Indeed! I did wonder why some thresholds were set as high as 50 =) Now knowing that it was pixels, not percentage, makes a lot of sense. Converted to a percentage based on `threshold` / `totalPixels`.

This also exposed a few other USD files that did not have their upAxis set. Updated those as well